docs/success-guide/command_line_options.md

```diff
@@ -15,6 +15,7 @@ The command line for user sync selects the set of users to be processed, specifi
 | ------------- |:-------------| 
 |   `--users all` |    All users listed in the directory are included.  |
 |   `--users group "g1,g2,g3"`  |    The named directory groups are used to form the user selection. <br>Users that are members of any of the groups are included.  |
+|   `--users mapped`  |    The same as --users group [...], but includes all directory groups defined in the configuration file. |
```

Comment:
The directory groups are not "defined" in the configuration file, and this construction makes it look like there might be an optional argument.  I suggest "The same as `--users group g1,g2,g3,...` where `g1,g2,g3,...` are all the directory groups specified in the configuration file group mapping."